docs/apis/avalanchego/public-api-server.md

```diff
@@ -35,12 +35,12 @@ Note: on Fuji Testnet, the URL is `wss://api.avax-test.network/ext/bc/C/ws`.
 
 The public API server supports all the API endpoints that make sense to be
 available on a public-facing service, including APIs for the
-[X-Chain](./apis/x-chain.md), [P-Chain](./apis/p-chain.md),
-[C-Chain](./apis/c-chain.md), and full archival for the Primary Network.
-However, it doesn't support [Index APIs](./apis/index-api.md), which includes
+[X-Chain](/reference/avalanchego/x-chain/api), [P-Chain](/reference/avalanchego/p-chain/api),
```

Comment:
I noticed that .md has been removed from all URLs and absolute paths are used. What is the reason for this? Have you read the benefits of relative path at the bottom of https://docusaurus.io/docs/markdown-features/links ?

Reply:
My original question is not answered: what is the reason to make this change where .md has been removed from all URLs and absolute paths are used?

Reply:
This change was made so that if a file is moved in the future, it will take only 1 search to locate every reference in the future and make that update

Reply:
How would leaving the .md reduce the amount of time it takes to update links?

Reply:
originally misunderstood the ask, thought their was some benefit to having the "/../../reference/etc" preserved. I added .md to all links and will keep it in mind for the changes going forward. all links will start from their root folder going folder which will make it easier to search for and correct if their are changes in the future.